packages/pluggableWidgets/bar-chart-web/src/BarChart.tsx

```diff
@@ -0,0 +1,62 @@
+import classNames from "classnames";
+import { createElement, ReactElement } from "react";
+import { ChartWidget, ChartWidgetProps } from "@mendix/shared-charts";
+import { usePlotChartDataSeries } from "@mendix/shared-charts/hooks";
+import { BarChartContainerProps } from "../typings/BarChartProps";
+
+const barChartLayoutOptions: ChartWidgetProps["layoutOptions"] = {
+    xaxis: {
+        zeroline: true,
+        fixedrange: true,
+        gridcolor: "#d7d7d7",
+        zerolinecolor: "#d7d7d7"
+    },
+    yaxis: {
+        fixedrange: true,
+        gridcolor: "#d7d7d7",
+        zeroline: true,
+        zerolinecolor: "#d7d7d7"
+    }
+};
```

Comment:
Please check [the previous docs](https://github.com/mendix/charts/blob/master/src/BarChart/utils/configs.ts) whether these values are correct. You might have missed something.